<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="Tree" name="_tree">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="ComputeBtn" name="_computeBtn">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Compute</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="AddGridBtn" name="_addGridBtn">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Add grid</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="MiniMap" name="_minimap">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="AddLabelBtn" name="_addLabelBtn">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Add label</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QTableView" name="_properties">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="6">
     <widget class="FullImage" name="_image">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>600</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="_statusBar"/>
  <action name="_actionGridAddRowTop">
   <property name="text">
    <string>Add row to top</string>
   </property>
   <property name="toolTip">
    <string>Add row to top</string>
   </property>
  </action>
  <action name="_actionSaveModel">
   <property name="text">
    <string>Save computation model</string>
   </property>
  </action>
  <action name="_actionLoadModel">
   <property name="text">
    <string>Load computation model</string>
   </property>
   <property name="toolTip">
    <string>Load computation model</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComputeBtn</class>
   <extends>QPushButton</extends>
   <header>helper.computebtn</header>
  </customwidget>
  <customwidget>
   <class>AddLabelBtn</class>
   <extends>QPushButton</extends>
   <header>helper.addlabelbtn</header>
  </customwidget>
  <customwidget>
   <class>AddGridBtn</class>
   <extends>QPushButton</extends>
   <header>helper.addgridbtn</header>
  </customwidget>
  <customwidget>
   <class>MiniMap</class>
   <extends>QLabel</extends>
   <header>helper.minimap</header>
  </customwidget>
  <customwidget>
   <class>FullImage</class>
   <extends>QLabel</extends>
   <header>helper.fullimage</header>
  </customwidget>
  <customwidget>
   <class>Tree</class>
   <extends>QTreeView</extends>
   <header>helper.tree</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
